### DISCUSSION TITLE
How do I write formulas better?

### DISCUSSION THREAD

**Discussion by JYunth** [tema6120/ForgetMeNot#59] [Q&A]
First of all, big fan of your app, dev. I use this app for Physics, chemistry and math, and I quickly realised I wasnt able to write formulas properly in plain text. Is there no support for richer formulas like latex, or am I not aware of it? If it is already implemented, I'm so sorry for being so ignorant.

**Comment by tema6120** [comment]
At this point plain text is only supported. Maybe in the future I will expand supported formats.
